.vscode/launch.json

```diff
@@ -51,7 +51,8 @@
                 "${file}"
             ],
             "env": {
-                "BICEP_TRACING_ENABLED": "true"
+                "BICEP_TRACING_ENABLED": "true",
+                "__EXPERIMENTAL_BICEP_REGISTRY_FQDN": "localhost:5000"

```

Comment:
Committed by mistake?

(It would probably be easiest to just exclude this file from the PR; the change on line 37 makes sense but isn't germane.)